src/content/docs/release-notes/infrastructure-release-notes/cloud-integration-release-notes/Agentless monitoring and standardized entity creation

```diff
@@ -0,0 +1,17 @@
+---
+title: Agentless monitoring for EC2 ,Azure VM and standardized entity creation for EC2, Azure VMs and GCP CE
+subject: Cloud integrations
+releaseDate: '2024-06-06'
+---
+
+Agentless monitoring for EC2 ,Azure VM and standardized entity creation across EC2, azure VMs, and GCP CE is now available. 
+
+## New 
+
+We updated our cloud host monitoring capabilities to help you monitor your Amazon EC2, Azure VM, and GCP CE instances more effectively. You'll now have access to:
+
+* Enhanced visibility into your Amazon EC2, Azure VM infrastructure without the need for complex setup or configuration
+* Simplified set-up that eliminates manual agent installations and configuration 
+* For a cloud resource(EC2, Azure VM, GCP CE) same entity is created when they are instrumented by both agents or as well as in agentless mode(API Polling - AWS/Azure/GCP, Cloudwatch Metric Streams integration, Azure Monitor integration)
```

Comment:
Hi @vinrelic, could you try to rewrite this last bullet? I'm a bit confused about what its saying.

Reply:
Is it possible to remove the last line? It does not add any value. Can you remove and publish?